src/truncate/truncate.component.js

```diff
@@ -0,0 +1,28 @@
+angular.module('patternfly.truncate').component('pfTruncate', {
+
+  bindings: {
+    content: '<',
+    limit: '<',
+    useWordBoundary: '<',
+    newlineLimit: '<',
+    removeExtraChars: '<',
+    expandCopy: '<',
+    collapseCopy: '<',
+    // When expandable is on, optionally hide the collapse link so text can only be expanded. (Used for toast notifications.)
+    expandable: '<',
+    //hideCollapse: '<',
+    // optional keywords to highlight using the `highlightKeywords` filter
+    keywords: '<',
```

Comment:
Currently it isn't used within this component... part of the original port from [Origin-Web-Common](https://github.com/cdcabrera/origin-web-common/blob/master/src/components/truncate-long-text/truncateLongText.js). Debating whether to include it, or simply allow some form of transcluded content